File: backend/chat-service/src/main/java/com/example/chatservice/member/dto/ChatRoomMemberDto.java
package com.example.chatservice.member.dto;

import com.example.chatservice.chat.dto.ChatRoomDto;
import lombok.AllArgsConstructor;
import lombok.Getter;

import java.util.List;

@Getter
@AllArgsConstructor
public class ChatRoomMemberDto {

    private Long chatRoomId;
    private List<MemberDto> chatRoomMembers;

    public ChatRoomMemberDto(ChatRoomDto chatRoom, List<MemberDto> members) {
        this.chatRoomId = chatRoom.getId();
        this.chatRoomMembers = members;
    }
}

File: backend/chat-service/src/main/kotlin/com/example/chatservice/member/dto/ChatRoomMemberDto.kt
package com.example.chatservice.member.dto

import com.example.chatservice.chat.dto.ChatRoomDto

class ChatRoomMemberDto(
    val chatRoomId: Long?,
    val chatRoomMembers: List<MemberDto>,
) {
    constructor(chatRoom: ChatRoomDto, members: List<MemberDto>) : this(chatRoom.id, members)
}
